include/mbedtls/cipher.h

```diff
@@ -524,15 +533,16 @@ int mbedtls_cipher_set_padding_mode( mbedtls_cipher_context_t *ctx, mbedtls_ciph
  * \brief           This function sets the initialization vector (IV)
  *                  or nonce.
  *
+ * \note            Some ciphers do not use IVs nor nonce. For these
```

Comment:
Pre-existing: `IVs` -> `IV's`.

Reply:
No, "IV's" means "of the IV" while "IVs" is more than one IV...

Reply:
Right, my bad!

Reply:
No worries